### DISCUSSION TITLE
Open Source Projects Showcase

### DISCUSSION THREAD

**Discussion by Shivansh-yadav13** [github/india#838] [Open Source]
**Let's promote some Open Source projects, and find new contributors!!**
- Would love to see new projects :smiley: 
- Some suggestion on how to make projects more beginner-friendly to get more contributors :thinking:

**Comment by Susmita-Dey** [comment]
> **Let's promote some Open Source projects, and find new contributors!!**
> 
> * Would love to see new projects 😃
> * Some suggestion on how to make projects more beginner-friendly to get more contributors 🤔

Here's my Open Source Project - [Moody](https://github.com/Susmita-Dey/Moody)
For making beginner-friendly projects and having more contributors, there should be well-documented steps in the README file and have some contribution guidelines and the repository maintainers must be active in order to manage and review the code of beginners and act in a friendly manner with the contributors.

**Comment by saanvi-ts** [comment]
> **Let's promote some Open Source projects, and find new contributors!!**
> 
> * Would love to see new projects 😃
> * Some suggestion on how to make projects more beginner-friendly to get more contributors 🤔

I would like to share our latest open source project [Materio Free Vuetify VueJS Admin Template](https://github.com/themeselection/materio-mui-react-nextjs-admin-template-free). 

It is a comprehensive & carefully crafted VueJS Admin Template for developing responsive web apps. With the help of this admin template, you can build any kind of web app. Such as,

- SaaS platforms
- Project management apps
- Ecommerce backends
- CRM systems
- Analytics apps
- Banking apps
- Education apps
- Fitness apps

**Features:**

- Pure Vue js
- Utilizes Vuex, Vue Router, Webpack
- 1 Dashboard
- Material Icons
- Basic cards
- Fully Responsive Layout
- Organized Folder Structure

You can check the demo here: [https://demos.themeselection.com/materio-vuetify-vuejs-admin-template-free/demo/dashboard](https://demos.themeselection.com/materio-vuetify-vuejs-admin-template-free/demo/dashboard).

**Comment by Yohjisakamoto** [comment]
## Tura — local, open-source coding agent

Tura is a local, open-source coding agent for developers who are tired of vague skill claims, token-saving extensions with no evidence, and agents that change repositories without understanding them first.

- Source and contribution entry point: https://github.com/Tura-AI/tura
- Website: https://turaai.net/
- Published benchmark, methodology, and artifacts: https://turaai.net/benchmark

In published DeepSWE v1.1 comparisons (20 tasks, 60 sessions, GPT-5.6 SOL High), Tura Direct used 83.5% fewer aggregate tokens than official Codex CLI High, with 65.0% vs 60.0% verifier success. Tura Balanced reached 80.0% success while using 49.6% fewer tokens. See the benchmark page for methodology, raw artifacts, and scope.

Disclosure: I’m affiliated with Tura-AI, and AI assistance was used to prepare this post; it was reviewed before publishing.
